Resources/doc/index.rst

```diff
@@ -0,0 +1,55 @@
+Getting Started With FOSRestBundle
+=====================================
+
+Installation
+------------
+
+Installation is a quick (I promise!) one-step process:
+
+1. :doc:`Setting up the bundle <1-setting_up_the_bundle.md>`
```

Comment:
You need to remove the file extension here (same with the other references in this file).